### DISCUSSION TITLE
[Enhancement] Azure Log example query change

### DISCUSSION THREAD

**Discussion by ehuggz** [grafana/grafana#37355] [Data Sources]
When creating new Azure Log Entries the automated example appears in the query window:
`//change this example to create your own time series query
//the table to query (e.g. Usage, Heartbeat, Perf)
\<table name\>

| where $__timeFilter(TimeGenerated)

//this is a macro used to show the full chart’s time range, choose the datetime column here

| summarize count() by <group by column>, bin(TimeGenerated, $__interval)

//change “group by column” to a column in your table, such as “Computer”. The $__interval macro is used to auto-select the time grain. Can also use 1h, 5m etc.

| order by TimeGenerated asc`

The references to the macros $__timeFilter(TimeGenerated) and $__interval are very helpful, but having them presented as uncommented code requires removing the example each time a new query is created. I often already have a Kusto Query copied into Clipboard from Azure ready to paste. I sometimes want to leave the references in place so I can add them to the query once pasted if I've been away from creating Grafana queries for a while.

**ASK**
Comment the entire set of example code rather than just the first line so that pasted code can run as is without error with reference code in place.

**Changed Example Query**

//QueryTemplate
//\<table name\>
//| where $__timeFilter(TimeGenerated)
//| summarize count() by <group by column>, bin(TimeGenerated, $__interval)
//| order by TimeGenerated asc

//Notes:
//1. Change \<table name\> to match your table for the query, (e.g. Usage, Heartbeat, Perf)
//2. The $__timeFilter(TimeGenerated) macro is used to show the full chart’s time range, replace the TimeGenerated datetime as required
//3. Change “group by column” in summarize statement to a column or columns in your table, such as “Computer”.
//4. The $__interval macro in bin() is used to auto-select the time grain. Can also use 1h, 5m etc.`
//5. Order by \<timeseries\> asc is required for Grafana Charting

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
